core/src/main/java/org/apache/accumulo/core/data/RowRange.java

```diff
@@ -45,8 +45,91 @@ public RowRange() {
    * @param endRow ending row; set to null for positive infinity
    * @throws IllegalArgumentException if end row is before start row
    */
-  public RowRange(Text startRow, Text endRow) {
-    this(startRow, true, endRow, true);
+  public static RowRange open(Text startRow, Text endRow) {
+    return create(startRow, true, endRow, true);
+  }
+
+  /**
+   * Creates a range of rows from startRow exclusive to endRow exclusive.
+   *
+   * @param startRow starting row; set to null for the smallest possible row (an empty one)
+   * @param endRow ending row; set to null for positive infinity
+   * @throws IllegalArgumentException if end row is before start row
+   */
+  public static RowRange closed(Text startRow, Text endRow) {
+    return create(startRow, false, endRow, false);
+  }
```

Comment:
This is incorrect. Closed means "inclusive", not "exclusive". Open means "exclusive". This is backwards throughout.

Reply:
Addressed in a527552 but please double check I got things correct